extensions/azurecore/src/extension.ts

```diff
@@ -160,10 +160,17 @@ export async function activate(context: vscode.ExtensionContext): Promise<azurec
 			ignoreErrors: boolean): Promise<azurecore.GetManagedDatabasesResult> {
 			return azureResourceUtils.getManagedDatabases(account, subscription, managedInstance, ignoreErrors);
 		},
-		getSqlServers(account: azurecore.AzureAccount,
+		async getSqlServers(account: azurecore.AzureAccount,
 			subscriptions: azurecore.azureResource.AzureResourceSubscription[],
 			ignoreErrors: boolean): Promise<azurecore.GetSqlServersResult> {
-			return azureResourceUtils.runResourceQuery(account, subscriptions, ignoreErrors, `where type == "${azurecore.azureResource.AzureResourceType.sqlServer}"`);
+			let serverResults = await azureResourceUtils.runResourceQuery(account, subscriptions, ignoreErrors, sqlServersQuery);
```

Comment:
Synapse workspaces aren't really servers though, I don't think we should be adding those here. And for these helper functions I think it's reasonable for now just to keep it using the previous query it was using before to just get all the sqlServer instances as they are. 

(it'll be up to the caller to do the additional lookups if they want to get the correct endpoints for the Synapse workspace pool instances)